docs/plugins.md

```diff
@@ -179,10 +179,6 @@ containing the Dockerfile.
   - Status: Enabled
   - Markdown help page is converted to a man page and ADD'ed into the built
     image in order to show a correct page when `atomic help` is called.
-- **add_content_sets**
```

Comment:
Have we replaced this docs when adding the new ICM plugin?

Reply:
Fixed in PR #1510